### PR TITLE
Put news relevancy behind feature flag

### PR DESCRIPTION
**Pull request by bsstoner**
https://duck.co/ia/view/news

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/spice/news/news.js [hunk 1]
```diff
@@ -6,61 +6,66 @@
...
-            "news",
-            "headline",
-            "headlines",
-            "latest",
-            "breaking",
-            "update",
-            "s:d",
-            "sort:date"
-        ];
+        var useRelevancy = DDG.opensearch.installed.experiment === "static_news" && DDG.opensearch.installed.variant === 'b',
```
Comment (bbraithwaite) [inline share/spice/news/news.js]: Wouldn't this switch off this logic for everybody but the `b` variant? Is that intended?

Comment (bsstoner) [inline share/spice/news/news.js]: crap, you're right, that's backwards.

**Comment by bbraithwaite** [comment]
👍

**Pull request merged**